Documents-Endpoint.md

```diff
@@ -249,19 +364,491 @@ Retrieve the full document bgu;11;2029
+
+Note that the `after` or `before` reference should also determine the depth of the insertion in the document's citation structure. The lowest structural level specified in the `after` or `before` reference is the level at which the contents of the XML `<dts:fragment>` will be inserted. In other words, the new segment will be taken to be a sibling to the segment specified in that reference.
+
+### POST Request Body
+
+The body of the POST request must be a properly formed `<TEI>` root node containing TEI compliant XML. See the further specifications under "Default Request and Response Format" above.
+
+Note that in most cases the actual text to be inserted is the __contents__ of the inner `<fragment>` element in the request body. The only exception to this rule is where no structural segment has yet been created for a document, in which case the entire `<TEI>` rootnode (along with all its contents) will be accepted as the initial form of the document.
+
+### POST Responses
+
+#### Status codes
+
+A successful POST request to the Documents endpoint should return the status code `201(Created)`.
+
+If a POST request is unsuccessful because of problems with the request content (i.e., parameters or request body), then the return status code should be `400(Bad Request)` or a custom status code in the 4XX series signaling a more specific error.
+
+A POST request should also return `409(Conflict)` if it would create a new initial form of a document for which some segments already exist (even if those segments are empty). This would happen when no `after` or `before` parameter is supplied, but at least one segment of the document already exists on the server (even if it contains only an empty string).
+
+A POST Documents request also may not result in a text segment whose reference identifier is the same as that of an existing segment. If, for example, a document already includes a segment "4.23.8", then a POST request which would result in a *second* segment "4.23.8" should fail and return `409(Conflict)`.
+
+#### Successful response headers
+
+The response headers after a successful POST Documents request should include a `Location` value. This should be the URL where the newly inserted text segment(s) can be retrieved via a GET request. The response should also include a `Content-type` header with a value of "application/tei+xml".
+
+
+The response should also include a `Link` header as detailed in the core documentation for the Documents endpoint. This `Link` gives URL references for the previous and next segments of the document, a URL for the document's navigation structure, and a URL for the document's Collections metadata. __If no previous or next segment exists, those parts of the `Link` header should give the URL for the newly created segment instead.__ If the request has created multiple segments, the `next` value for the `Link` header should be the last created segment, and the `previous` value should be the first created segment.
```

Comment:
"__If no previous or next segment exists, those parts of the `Link` header should give the URL for the newly created segment instead.__ " I don't understand this. Why would we return prev/next links in this case?

Reply:
We agreed to remove this use of the Link header.